app/views/converts/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<% title "convert" %>
+<% content_for(:header) do %>
+  <h2>convert splits to other cool formats</h2>
+<% end %>
+<article>
+  <form id="convert" action="/api/v4/convert" method="post" enctype="multipart/form-data">
+    <div class="form-group col-sm-4">
+      <input type="file" name="file" />
+    </div>
+    <div class="form-group col-sm-2">
+      <select name="format">
+        <% (Run.programs - [Llanfair]).each do |prgrm| %>
+          <option value="<%= prgrm.to_sym %>"><%= prgrm.to_s %></option>
+        <% end %>
+        <option value="json">JSON</option>
+      </select>
+    </div>
+    <div class="form-group col-sm-2">
+      <input type="checkbox" name="historic" id="historic" />
+      <label for="historic">Incldue History?</label>
+    </div>
+    <div class="form-group col-sm-2">
+      <input type="submit" value="convert" />
+    </div>
+  </form>
+</article>
```

Comment:
we use slim instead of erb for most html templates, can we change this file to use it